manager/allocator/network.go

```diff
@@ -587,8 +587,8 @@ func (a *Allocator) allocateServices(ctx context.Context, existingAddressesOnly
 			continue
 		}
 
-		if err := a.allocateService(ctx, s); err != nil {
-			log.G(ctx).WithError(err).Errorf("failed allocating service %s during init", s.ID)
+		if err := a.allocateService(ctx, s, existingAddressesOnly); err != nil {
```

Comment:
nit: Please add a comment on how existingAddressesOnly relates to the restart flag which is an arg for allocateService()

Reply:
Did you mean to make this change ? I don't see it in the patch @fcrisciani

Reply:
the name of the variable is changed inside the allocateService not here, I kept the original name `existingAddressesOnly`